### DISCUSSION TITLE
How to use `j` `k` in eww input place?

### DISCUSSION THREAD

**Discussion by idlip** [meow-edit/meow#440] [Q&A] [answered]
First of all, i just gave a try for modal package, and meow seems to be my fit, as it plays really well with vanilla emacs.

But I faced an issue right now, I see `j` and `k` are bound to `meow-prev` and `meow-next` in eww-mode, but in the eww text input area I cannot use letter `j` or `k`, as it does `meow-prev` and next

Eg you can try eww with this site :

https://lite.duckduckgo.com/lite/?q=emacs

**Comment by idlip** [comment, marked as answer]
Ig it was too haste of me to jump on this, by default with eww-mode meow open in `motion-mode` so getting into insert kinda solves it.

I'm still new to meow, so should explore more thoroughly.

Ig i will mark it as solved, any comments or suggestion are welcome tho

**Reply by DogLooksGood** [reply]
It depends on what behavior you want
1. Do you still want a leader key at space bar?
2. Do you mind toggling insert state or you want it be more like without modal editing?

Some solutions are, creating a dedicated state with `meow-define-state` for eww, tweaking the keymap with a buffer-local post-command-hook, ec.

**Reply by idlip** [reply]
Manually toggling is fine for now. Will see if it gets annoying (will change by then)

Will explore more on `meow`.
Thank you for informing.
